frontend/public/components/edit-yaml.jsx

```diff
@@ -43,7 +43,12 @@ const stateToProps = ({k8s, UI}) => ({
  * This component loads the entire Ace editor library (~100kB) with it.
  * Consider using `AsyncComponent` to dynamically load this component when needed.
  */
-/** @augments {React.Component<{obj?: any, create: boolean, kind: string, redirectURL?: string}>} */
+/** @augments {React.Component<{
+ *    obj?: any,
+ *    create: boolean,
+ *    kind: string,
+ *    redirectURL?: string,
+ *    resourceObjPath?: (obj: k8sKind, ...extraArgs: any[]) => string } */
```

Comment:
This relates to https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html

I'd prefer to keep the JSDoc tag's value on one line to avoid potential issues with TS comment parsing.

(This is a non-TypeScript file that should be eventually transformed to TS and moved to appropriate package.)

Reply:
Also, why do we need the `...extraArgs: any[]` part, given how it's called in the code below?

The signatures of `props.resourceObjPath` vs `resourceObjPath` helper should be aligned:

```ts
resourceObjPath?: (obj: k8sKind, objRef: string) => string
```

Reply:
Thanks , looks better with same signature :+1: fixing !